src/art.cpp

```diff
@@ -1359,10 +1372,10 @@ int art_fuzzy_search(art_tree *t, const unsigned char *term, const int term_len,
         std::sort(nodes.begin(), nodes.end(), compare_art_node_score);
     }
 
-    long long int time_micro = microseconds(std::chrono::high_resolution_clock::now() - begin).count();
+    //long long int time_micro = microseconds(std::chrono::high_resolution_clock::now() - begin).count();
```

Comment:
Did you intend to leave this line commented out?

Reply:
Yes, this must have been commented out already but it was not.